__tests__/app.test.js

```diff
@@ -131,6 +131,30 @@ describe("GET /api/articles/:article_id/comments", () => {
   });
 });
 
+describe("POST /api/articles/:article_id/comments", () => {
+  test("POST 201 adding successfully a comment for an article", () => {
+    const newComment = {
+      username: "icellusedkars",
+      body: "The path of the righteous man is beset on all sides by the inequities of the selfish and the tyranny of evil men...",
+    };
+
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const postedComment = body.newComment[0];
```

Comment:
We need to send back the comment as an object on the key of newComment. We should not send back the array